### PR TITLE
Add support for markerNames  parameter 

### PR DESCRIPTION
**Pull request by AllenBW**
Also: 
* Adds tests
* Updates documentations
* :taco: :dancer:
* closes https://github.com/huston007/angular-gettext-cli/issues/10

### CODE REVIEW

File: bin/gettext [hunk 1]
```diff
@@ -27,7 +27,9 @@ function objToUpperCase(obj) {
 
         obj[key] = obj[keys[i]];
     }
-
+    if(obj.markerNames){
```
Comment (himdel) [inline bin/gettext]: `if (obj.markerNames) {` ;)

**Comment by himdel** [comment]
This also enables the traditional use of both `_` and `N_` , which does not seem to be possible without this..

File: bin/gettext (hunk 1 above)
Comment (AllenBW) [inline bin/gettext]: OH SNAP good catch on the spacing 💟

**Comment by huston007** [comment]
Awesome, thanks!

**Comment by huston007** [comment]
Published as `angular-gettext-cli@1.2.0`

**Pull request merged**